modules/site/src/app/examples/button/button-basic-usage.component.spec.ts

```diff
@@ -1,4 +1,11 @@
-import {beforeEach, beforeEachProviders, describe, expect, it, inject,} from '@angular/core/testing';
+import {
+  beforeEach,
+  beforeEachProviders,
+  describe,
+  expect,
+  it,
+  inject,
```

Comment:
Add [codelyzer](https://github.com/mgechev/codelyzer) to a `tslint` process? Gives developers warnings about formatting inconsistencies, and lets them decide how to make their files match it, (versus a script that does things like add trailing commas, break lines at harder-to-read places, etc.)